### PR TITLE
Issue #20: personalizing script with user location and contact name (WIP)

### PR DESCRIPTION
**Pull request by tinyels**
This is "working" but we have not gotten the Script component under test and would like to do that before completing the issue.

### CODE REVIEW

**Comment by nickoneill** [comment]
It looks like the script test file is empty - so Jest is complaining about there being no tests in what is supposed to be a test file.

If you want to add tests here, try copying one of the snapshot tests from `Call.test.tsx` and replacing it with the script component, snapshot makes sense for this type of test.

File: src/components/call/CallPageContainer.tsx [hunk 1]
```diff
@@ -68,7 +71,8 @@ const mapStateToProps = (state: ApplicationState, ownProps: OwnProps): StateProp
   return {
     callState: state.callState,
     currentIssue: currentIssue,
-    splitDistrict: state.locationState.splitDistrict
+    splitDistrict: state.locationState.splitDistrict, // todo: just pass locationState and remove separate splitDistrict
```
Comment (nickoneill) [inline src/components/call/CallPageContainer.tsx:76]: Good `todo` here, definitely doesn't need to be refactored with the other changes.

**Comment by nickoneill** [comment]
Re: the funky test result we discussed in slack, here was the proposed solution for the moment:

Use a similar method to `export const ContactOfficesTranslatable = translate()(ContactOffices);` that appears in `ContactOffices.tsx`.

Requiring full instances of the i18n-react for each test isn't ideal but I'm not quite certain how to work around the issue at the moment. We can refactor at a future time.

**Comment by nickoneill** [comment]
Looking good @tinyels, tests are not complaining anymore about translations! 🙃

The currently failure is because the script section isn't rendering anything. You need an issue + contact to render the script section, I would grab the test object that's built in `ContactOffices.test.tsx` and try using that.

**Comment by tinyels** [comment]
Thanks @nickoneill, I committed this so that @lnathanson can finish up the tests. 😄

**Comment by nickoneill** [comment]
Perfect, thanks!

**Comment by nickoneill** [comment]
Hi all - I'm going to merge this in and add a snapshot test to verify that this component doesn't change unexpectedly. If you have other plans for more concrete tests of functionality, please send a new PR!

Thanks for your help on this!

**Pull request merged**